src/shims-vue.d.ts

```diff
@@ -0,0 +1,5 @@
+// needed for unit test files to find component modules
+declare module '*.vue' {
+  import Vue from 'vue'
+  export default Vue
+}
```

Comment:
The symptom of not having this file is a warning in the unit test files that the components under test can't be found.